BotRoyaleUnity/Assets/Scripts/BuildRobot.cs

```diff
@@ -13,7 +13,7 @@ public class BuildRobot : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        //BuildSampleRobots();
+        BuildSampleRobots();
```

Comment:
It may be time to either split this out into it's own script or make it a bool setting on this class, since some scenes need these robots for testing and some scenes dont